app/models/census/api/definitions.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Census
+  module API
+    # Container module for Census API constants
+    module Definitions
+      extend ActiveSupport::Concern
+
+      DOCUMENT_TYPES = %w(dni nie passport).freeze
```

Comment:
I'm not sure, but when you define constants in a concern, all modules including it will have the same constants? I saw that you are refering to them as `Census::API::Definitions::STATES`, wouldn't `STATES` be enough, as is a local constant? :thinking: